openvdb/openvdb/python/CMakeLists.txt

```diff
@@ -105,6 +102,14 @@ if(USE_NUMPY)
   list(APPEND OPENVDB_PYTHON_REQUIRED_COMPONENTS NumPy)
 endif()
 
+# scikit-build doesn't yet support FindPython, but this workaround was
+# supplied by one of the scikit-build devs. See
+# https://github.com/pypa/cibuildwheel/issues/727 for details.
+if(SKBUILD)
+    set(Python_EXECUTABLE "${PYTHON_EXECUTABLE}")
```

Comment:
Is this still needed?

Reply:
Really not sure, it's been a long time since this was put in but I recall it being necessary to get the bindings to build. I'll double check now and report back.

Reply:
Doesn't look like it anymore. Removing.